tests/integration/tier0/destructive/offline-system-conversion/test_offline_system_analysis.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from conftest import TEST_VARS
+
+
+@pytest.mark.test_offline_system_analysis
+def test_offline_system_analysis(shell, convert2rhel):
```

Comment:
Once the https://github.com/oamg/convert2rhel/pull/1194 is merged, we can use the satellite registration as a fixture (https://github.com/oamg/convert2rhel/pull/1194/files#diff-e1453e6b1978b5d69918339a6c43ae392039803329713923ce6d0a8d37a7e7dcR7). Also it will be possible to run on Oracle systems

Reply:
The #1194 was merged (satellite test - https://github.com/oamg/convert2rhel/blob/main/tests/integration/tier0/destructive/conversion-method/test_satellite.py#L7)